### PR TITLE
Outfit data / Initial Styling Completed

### PR DESCRIPTION
**Pull request by dprevoznik**
Hey everyone. Please take a look at my code up for review. I added local storage manipulation to pull in outfit data and have my initial styling set up using bulma.

### CODE REVIEW

File: src/components/productDetails/list.jsx [hunk 1]
```diff
@@ -1,41 +1,70 @@
...
+
+  if (listName === "Outfit") {
+    products = outfits;
+    productsImages = outfitsImages;
+  }
   //HANDLE FETCHING DATA ON AFTER FIRST RENDER
   useEffect(() => {
-    fetchRelatedDataAsync(1);
+    if (listName === "Related") {
+      fetchRelatedDataAsync(4);
```
Comment (mayannechess) [inline src/components/productDetails/list.jsx:30]: We'll all have to figure out how to move away from hard-coding product IDs as we have a better idea about how the store will be initialized.

File: src/components/productDetails/list.jsx [hunk 2]
```diff
@@ -1,41 +1,70 @@
...
   return (
-    <div>
+    <div class="columns">
+      {listName === "Outfit" ? (
+        <AddToOutfitCard
+          pageProduct={pageProduct}
+          onClickButton={onClickButton}
+        />
+      ) : (
+        <div></div>
```
Comment (mayannechess) [inline src/components/productDetails/list.jsx:66]: Do we have a group preference on empty div vs null in conditional rendering? Are there specific use cases for one or the other?

File: src/components/productDetails/outfitHelpers.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    console.log("no outfits saved in local storage");
+  } else {
+    let OutfitIds = Object.values(localOutfit);
+    fetchImages(OutfitIds)
+      .then(responseArr => {
+        let dataArray = responseArr.map(response => response.data);
+        setOutfitsImages(dataArray);
+      })
+      .catch(err => {
+        console.log("err trying to fetch all related product data: ", err);
```
Comment (mayannechess) [inline src/components/productDetails/outfitHelpers.js:18]: Might be useful to distinguish this error message from the one below

File: src/components/productDetails/outfitHelpers.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+export function addToOutfit(id) {
+  let localOutfit = JSON.parse(window.localStorage.getItem("outfit"));
+  if (localOutfit === null) {
+    // Local storage is empty, so assign new object
+    window.localStorage.setItem("outfit", JSON.stringify({ [id]: id }));
+  } else {
+    // Combine existing storage with new item / override
+    let newLocalOutfit = Object.assign({}, localOutfit, { [id]: id });
+    window.localStorage.setItem("outfit", JSON.stringify(newLocalOutfit));
+    console.log('outfits in local storage after add: ', window.localStorage.getItem('outfit'))
```
Comment (mayannechess) [inline src/components/productDetails/outfitHelpers.js:50]: Nice logging throughout. Will be helpful debugging

**Review by mayannechess — APPROVED** [review]
It's great that you've gotten so much done on this. It seems like you have great habits around modularization and clarifying through features like destructuring. Going forward, the team should figure out collectively how to stop hard-coding IDs... and maybe discuss null vs. empty div?

Your local storage manipulation will be a great example for other team members who need to use this feature.

File: src/components/productDetails/list.jsx (hunk 1 above)
Comment (dprevoznik) [reply src/components/productDetails/list.jsx:30]: Yes i agree. Something we will need to keep in mind.

File: src/components/productDetails/outfitHelpers.js (hunk 3 above)
Comment (dprevoznik) [reply src/components/productDetails/outfitHelpers.js:18]: Good idea. Will keep in mind.

**Pull request merged**